src/helpers/express_session.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express')
+const app = express()
+const session = require('express-session')
+
+app.use(session({
+    secret:'urlShortner',
+    saveUninitialized:true,
+    resave:true
+}))
+var userId;
```

Comment:
Hey bro, kindly point the line that extends the session length indefinitely

Reply:
session length by default is indefinite,unless i change it

Reply:
if u need me to specifically set it,all i need to do is set makAge,but by default it is nulll which means it dosent expire